src/test/java/com/google/javascript/gents/multiTests/module_imports/export_sideeffect.ts

```diff
@@ -1,2 +1,5 @@
-export {};
+export {
+  //  Empty export added to coerce the file to be a module, since it does not
```

Comment:
Unfortunately, a newline is added after the comment (either by clang or gents), so this can't be on one line; explicitly making it multi-line, which is prettier than:
```javascript
export {/* gents: force this file to be an ES6 module (no imports or exports) */
};
```

@gmoothart: Unfortunately the comment can't be added before the node; otherwise, the comment gents separated from the node by clang-format, which thinks it's a file overview comment.